enricher/api/src/main/java/io/fabric8/maven/enricher/api/BaseEnricher.java

```diff
@@ -35,11 +50,11 @@
 
     private final EnricherConfig config;
     private final String name;
-    private EnricherContext buildContext;
+    private MavenEnricherContext buildContext;
 
     protected Logger log;
 
-    public BaseEnricher(EnricherContext buildContext, String name) {
+    public BaseEnricher(MavenEnricherContext buildContext, String name) {
```

Comment:
The BaseEnricher is the base class of almost every enricher, so if this depends on the `MavenEnricherContext` directly then all enrichers are _still_ coupled to Maven. Actually I believe only the `EnricherContext` should be added here.